components/slack/sources/new-star-added/new-star-added.mjs

```diff
@@ -23,8 +23,12 @@ export default {
   },
   methods: {
     ...common.methods,
-    getSummary() {
-      return "New star added";
+    getSummary(event) {
+      let type = (event.item.type === "im")
```

Comment:
Could you please add some comments about these operations? Looks like you are normalizing the data for some reason, it would be great if you could comment your thoughts here to help a future maintenance

Reply:
It's just to show the type in the summary

Reply:
But it would be great if could comment on why are you slicing and using charAt. Just to help if some other people would maintain the code. For example: What exactly are you slicing? What was the previous data? What is the result of your conversion?